frontend/src/component/common/Input/Input.tsx

```diff
@@ -26,7 +25,6 @@ const Input = ({
     className,
     value,
     onChange,
-    InputProps,
```

Comment:
I noticed this broke passing `InputProps` to our `TextField`. This meant things like:
https://github.com/Unleash/unleash/blob/8470fc0e215558e57afa7242988afe7c72aa0cfb/frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantModal/EnvironmentVariantModal.tsx#L387-L397
Wouldn't work. 
It also means that usage of `InputProps` will now start working correctly for all input fields using this component, so we may see unrelated changes in other places where the fields will now behave like they should.